package.json

```diff
@@ -27,5 +44,21 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "@testing-library/react": "^9.1.3",
+    "eslint-config-airbnb": "^17.1.1",
+    "eslint-plugin-babel": "^5.3.0",
+    "eslint-plugin-import": "^2.18.0",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
+    "eslint-plugin-react": "^7.14.2",
+    "eslint-plugin-react-hooks": "^1.6.1",
+    "husky": "^3.0.3",
+    "lint-staged": "^9.2.1",
+    "expect": "^24.9.0",
```

Comment:
When you use `Jest`, I think you might not need to install `expect` explicitly ?

Reply:
I forgot to remove it while rebasing, thanks!